android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerModule.kt

```diff
@@ -338,9 +337,11 @@ class RNGestureHandlerModule(reactContext: ReactApplicationContext?) : ReactCont
   }
 
   @ReactMethod
-  fun attachGestureHandler(handlerTag: Int, viewTag: Int) {
+  private fun attachGestureHandler(handlerTag: Int, viewTag: Int, useDeviceEvents: Boolean) {
```

Comment:
Why did you mark it as private?

Reply:
Fixed in 18a1898846f12ca4cf801add4bfa87b3df3c15a1. At first it was a helper method invoked in the other 2, but then I removed them and forgot to make this one public.